File: Domain/Inquiry.cs
namespace Domain;
public class Inquiry
{
    public int Id { get; set; }
    public int Width { get; set; }
    public int Height { get; set; }
    public int Weight { get; set; }
    public DateOnly Date { get; set; }
    public int SourceAddressId { get; set; }
    public Address SourceAddress { get; set; }
    public int DestinationAddressId { get; set; }
    public Address DestinationAddress { get; set; }
    public string Priority { get; set; }
    public bool DeliveredOnWeekend { get; set; }
}

File: DHL/client/src/models/inquiry.ts
export interface IInquiry{
    id: number;

    width?: number;
    height?: number;
    weight?: number;
    date?: Date;
    
    sourceStreetName?: string;
    sourceStreetNumber?: string;
    sourceFlatNumber?: string;
    sourceZipCode?: string;
    sourceCity?: string;

    destinationStreetName?: string;
    destinationStreetNumber?: string;
    destinationFlatNumber?: string;
    destinationZipCode?: string;
    destinationCity?: string;

    submitted?: boolean;
    offers?: number[];
    priority?: string;
    deliveredOnWeekend?: boolean;
}

export function getEmptyInquiry (){
    const inquiry = 
    {
        id: -1,
        width: undefined,
        height: undefined,
    }
    return inquiry;
}